libraries/AP_GPS/AP_GPS_MAV.cpp

```diff
@@ -114,7 +114,7 @@ void AP_GPS_MAV::handle_msg(const mavlink_message_t *msg)
             mavlink_msg_hil_gps_decode(msg, &packet);
 
             state.time_week = 0;
-            state.time_week_ms  = packet.time_usec/1000;
```

Comment:
This doesn't look right.

Reply:
oh, is that 64bit? Then it should be removed.